website/content/docs/upgrading/instructions/testing-experimental-raft-backend.mdx

```diff
@@ -0,0 +1,420 @@
+---
+layout: docs
+page_title: Testing the Experimental WAL LogStore Backend
+description: >-
+  Consul 1.15 introduced a new experimental storage backend option. Learn how to
+  configure and test it out in a safe way.
+---
+
+# Testing the Experimental WAL LogStore Backend
+
+## Introduction
+
+Consul 1.15 introduced a new experimental storage backend. This guide explains
+how to configure it and test it out safely. The new backend is called `wal` in
+configuration.
+
+WAL is a acronym for "Write-Ahead Log". We called it this because it implements
+a traditional log with rotating, append-only log files. The current `LogStore`
+uses BoltDB which is a copy-on-write BTree which is less optimized for
+append-only workloads.
+
+~> The `wal` backend is considered **experimental** in Consul 1.15. Please test
+it safely in pre-production first and verify on a subset of servers in
+production using this guide before fully enabling it on all servers.
+
+## Why build a new backend?
```

Comment:
```suggestion
### WAL versus BoldDB
```
I think this is still part of the introduction. In fact, most of the information in this section, I would argue, isn't documentation and would be better suited to a blog article.

Reply:
I actually had `// TODO should this be in a blog?` right up until I posted the PR on this section. My main reasoning not to was just that I think this needs to live _somewhere_ and don't think a blog will be ready on release day so wanted to land it somewhere at least initially so the content is available.

I'll go back and look at timing to see if we can make a blog happen in time.